lib/releases.sh

```diff
@@ -42,6 +45,8 @@ else
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
 fi
 
+export BMORELEASE="${BMORELEASE:-$(get_latest_release "${BMORELEASEPATH}" "v0.1.")}"
+
 CAPIBRANCH="${CAPIBRANCH:-${CAPIRELEASE}}"
```

Comment:
most probably you have to set BMOBRANCH here same way since you are not setting any default in common.sh and this file is not sourced later when needed.

Reply:
Indeed, added that, lets see.

Reply:
Its working it seems. Thanks